solr/docker/templates/Dockerfile.body.template

```diff
@@ -72,6 +72,8 @@ RUN set -ex; \
 RUN set -ex; \
     apt-get update; \
     apt-get -y --no-install-recommends install acl lsof procps wget netcat gosu tini jattach; \
+    wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/1.17/gosu-amd64"; \ 
+    chmod +x /usr/local/bin/gosu; \
```

Comment:
It is a known fact that our version of ubuntu offers a slightly older `gosu` binary based on an older golang. However, there are no known attack vectors known to us how an external attacker might abuse this.

The fix presented here is not acceptable, as the image must work both for amd and arm architectures.

A better solution is to wait until we bump our default Ubuntu version in the docker image. Which I suppose could happen soon?

Reply:
We should definitely bump it for 10x